ingestion/functions/parsing/brazil_amapa/amapa.py

```diff
@@ -15,19 +15,19 @@
             'common/python'))
     import parsing_lib
 
-_DATE_CONFIRMED_INDEX = 0
-_HEALTHCARE_WORKER_INDEX = 1
-_PREEXISTING_CONDITIONS_INDEX = 2
-_METHOD_CONFIRMATION_INDEX = 3
-_CONFIRMATION_INDEX = 4
-_GENDER_INDEX = 5
-_NOTES_BAIRRO_INDEX = 6
-_MUNICIPALITY_INDEX = 7
-_AGE_INDEX = 8
-_ETHNICITY_INDEX = 9
-_NOTES_INDIGENOUS_GROUP_INDEX = 10
-
-commorbidities = {
+_DATE_CONFIRMED = "dataNotificacao"
+_HEALTHCARE_WORKER = "profissionalSaude"
+_PREEXISTING_CONDITIONS = "comorbidades"
+_METHOD_CONFIRMATION = "tipoTeste"
+_CONFIRMATION = "resultadoTeste"
+_GENDER = "sexo"
+_NOTES_BAIRRO = "bairro"
+_MUNICIPALITY = "municipio"
+_AGE = "idade"
+_ETHNICITY = "racaCor"
+_NOTES_INDIGENOUS_GROUP = "etniaIndigena"
+
+_COMORBIDITIES_MAP = {
```

Comment:
This is a great change, it will be more reliable if the columns get reordered in the data.

Reply:
yep, exactly